toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -51,6 +52,7 @@ import kotlinx.coroutines.launch
  * @param viewpointOperation a [SceneViewpointOperation] that changes this SceneView to a new viewpoint
  * @param graphicsOverlays the [GraphicsOverlayCollection] used by this composable SceneView
  * @param sceneViewProxy the [SceneViewProxy] to associate with the composable SceneView
+ * @param sceneViewInteractionOptions a [SceneViewInteractionOptions] that configures the interaction options for the composable SceneView
```

Comment:
to be consistent with the corresponding doc on composable MapView:
```suggestion
 * @param sceneViewInteractionOptions the [SceneViewInteractionOptions] used by this composable SceneView
```